release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -86,6 +86,68 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-mirror-v2_{context}"]
```

Comment:
This note has a lot of subheadings which is distracting for surrounding notes. I suggest maybe adding each subheading as a sub-bullet point and let the feature docs do the describing.